### PR TITLE
Feature/docker xenial

### PR DESCRIPTION
**Pull request by tomereli**
To make sure old GCC versions are tested, add build-in-docker-xenial
step to .gitlab-ci.yml.

### CODE REVIEW

**Review by VladyslavTupikin — APPROVED** [review]

File: tools/docker/builder/xenial/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,28 @@
+FROM ubuntu:xenial
```
Comment (arnout) [inline tools/docker/builder/xenial/Dockerfile]: In the other one we call it ubuntu:18.04, so
```suggestion
FROM ubuntu:16.04
```

File: tools/docker/builder/xenial/Dockerfile [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+    cmake \
+    curl \
+    gcc \
+    git \
+    libjson-c-dev \
+    libncurses-dev \
+    libnl-genl-3-dev \
+    libreadline-dev \
+    libssl-dev \
+    libssl-dev \
```
Comment (arnout) [inline tools/docker/builder/xenial/Dockerfile]: Twice?
```suggestion
    libssl-dev \
```

File: .gitlab-ci.yml [hunk 3]
```diff
@@ -111,6 +110,12 @@ build-in-docker-alpine:
   only:
     - schedules
 
+build-in-docker-xenial:
+  extends: build-in-docker
+  image: prplfoundationinc/prplmesh-builder:xenial
```
Comment (arnout) [inline .gitlab-ci.yml]: Please build the image in CI as well, and use the new naming.

**Review by vitalybu — APPROVED** [review]

File: .gitlab-ci.yml [hunk 4]
```diff
@@ -111,6 +110,12 @@ build-in-docker-alpine:
...
+  # only:
+  #   - schedules
```
Comment (arnout) [inline .gitlab-ci.yml]: Please don't leave in this commented-out stuff.

File: .gitlab-ci.yml [hunk 5]
```diff
@@ -250,12 +255,12 @@ run-certification-tests:
...
-            echo "Deploying to $DEVICE_UNDER_TEST"
-            tools/deploy_ipk.sh --certification-mode $DEVICE_UNDER_TEST build/$DEVICE_UNDER_TEST/prplmesh.ipk
-        fi
-      - /easymesh_cert/run_test_file.sh -v -o logs -d $DEVICE_UNDER_TEST $TESTS_TO_RUN
+    - |
+      if [ "$DEVICE_UNDER_TEST" != prplmesh ] ; then
+          echo "Deploying to $DEVICE_UNDER_TEST"
+          tools/deploy_ipk.sh --certification-mode $DEVICE_UNDER_TEST build/$DEVICE_UNDER_TEST/prplmesh.ipk
+      fi
+    - /easymesh_cert/run_test_file.sh -v -o logs -d $DEVICE_UNDER_TEST $TESTS_TO_RUN
```
Comment (arnout) [inline .gitlab-ci.yml]: Please do the formatting corrections in a separate commit. (We should probably run yamllint on all yaml files...)

**Review by arnout — APPROVED** [review]
